packages/types/src/node-protocol.ts

```diff
@@ -87,9 +90,11 @@ export namespace Node {
 
   export type WithdrawParams = {
     multisigAddress: string;
+    recipient?: string;
     amount: BigNumber;
   };
   export type WithdrawResult = {
+    recipient: string;
```

Comment:
@snario Why is it required to have a `recipient` here and not on the `WithdrawParams` type?

Reply:
In the params it can go to a default value but in the result it gives you the one it used (either the param or the default)